.github/pull_request_template.md

```diff
@@ -6,22 +6,26 @@
 
 ## General information
```

Comment:
nit: I believe the "Description" and "General information" headers can be a single one called "Description".

No change in content though.